src/lib/components/Swap/Swap.fixture.tsx

```diff
@@ -71,6 +71,7 @@ function Fixture() {
       defaultOutputAddress={optionsToAddressMap[defaultOutput]}
       defaultOutputAmount={defaultOutputAmount}
       tokenList={tokens}
+      onConnectWallet={() => console.log('onConnectWallet')}
```

Comment:
is this intentional?

Reply:
Yes - this way it will show as a button in our cosmos viewer, and _something_ will happen when you press it.

Reply:
Removed this.

Reply:
jk I just documented this with a comment.